### ISSUE TITLE
Change this to a set of files for pull requests on the lessons?

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/js-javascripting-study#2]
I think Jeff and I discussed this but about learnyounode

**Comment by jrhorn424** [comment]
Yes. It should be a blank set of files ready for forking and cloning, and suitable for pull-request diffing.

Any preferences on names? I figure order doesn't matter much. Naming after each workshop lesson is obvious, but it isn't obvious whether adding numeric prefixes for order would help students.

**Comment by jrhorn424** [comment]
If we pursue this, it makes #1 obsolete.

**Comment by gaand** [comment]
Should files be named NN-obvious-name.js, e.g. 00-introduction.js, 01-variables.js?  With instruction?  For this lesson it's not necessary and possibly confusing (file names are suggested), but for learnyounode might it be helpful (file names not suggested)?

**Comment by jrhorn424** [comment]
I believe we used this approach in the nodeschool assignment repo. I'd like to go with whatever is less confusing. May not know that until we try it. ;)

**Comment by gaand** [comment]
Let's try NN-obvious-name.js.  At least they sort sensibly :-P.

**Comment by jrhorn424** [comment]
:+1:

**Comment by jrhorn424** [comment]
@gaand do you still believe this should be a required pull request?

I can setup a sample structure in any case. Or I can let them structure however they want.

Not sure what the advantage of reviewing any PRs would be over diagnosis as a separate task.

**Comment by gaand** [comment]
@jrhorn424 Pull request not required.  If students want feedback then PR?

**Comment by jrhorn424** [comment]
OK, I agree. Next cohort, let's have the file structure in place in case students wish to submit a PR for feedback.

**Comment by jrhorn424** [comment]
Not sure this is necessary any longer. The introduction at least has you make a folder and add files.

<img width="622" alt="__teaching_wdi_boston_js-javascripting-study" src="https://cloud.githubusercontent.com/assets/388761/15482609/84f0e64c-20fe-11e6-976b-447d335d1179.png">

**Comment by gaand** [comment]
Can we incorporate that into using this repo and then requiring a pull request?

If the instructions include file names, then that should be straightforward.

**Comment by jrhorn424** [comment]
I agree.
